docs/conversion/sakai_22_mysql_conversion.sql

```diff
@@ -30,3 +30,20 @@ ALTER TABLE GB_GRADEBOOK_T ADD COLUMN COMPARING_DISPLAY_ALLITEMS BIT(1) DEFAULT
 ALTER TABLE GB_GRADEBOOK_T ADD COLUMN COMPARING_RANDOMIZEDATA BIT(1) DEFAULT FALSE NOT NULL;
 -- End SAK-44967
 
+-- SAK-43155
+ALTER TABLE ASN_ASSIGNMENT ADD IS_REQUIRED_ESTIMATE CHAR(1) NULL;
```

Comment:
If this is a boolean value, it should have a default and not be allowed to be nullable.

Reply:
Done, but bit type is not possible since there's a constraint that makes sure this column only contains char type values into 0, 1.